public/index.html

```diff
@@ -23,19 +23,116 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>React App</title>
+    <style type='text/css'>
```

Comment:
can also probably move the styling to the bottom if we aren't doing anything for offline stuff